resolver/GlobalPass.cc

```diff
@@ -82,7 +82,12 @@ bool resolveTypeMember(core::GlobalState &gs, core::ClassOrModuleRef parent, cor
     if (!sym.data(gs)->derivesFrom(gs, core::Symbols::Class()) && myVariance != parentVariance &&
         myVariance != core::Variance::Invariant) {
         if (auto e = gs.beginError(myTypeMember.data(gs)->loc(), core::errors::Resolver::ParentVarianceMismatch)) {
-            e.setHeader("Type variance mismatch with parent `{}`", parent.show(gs));
+            e.setHeader(
+                "Type variance mismatch with parent `{}`; child `{}` should be `{}` or invariant, but it is `{}`",
```

Comment:
If the class has multiple type members, this error message does not mention the name of which of the type members is incompatible. Do you think you could add that information somewhere? It should be as simple as adding `name.show(gs)` somewhere in the error message header.